TradingBot/Models/TradingAction.cs

```diff
@@ -129,7 +130,7 @@ public static TradingAction MarketSell(TradingSymbol symbol, decimal quantity, D
             Id = Guid.NewGuid(),
             CreatedAt = createdAt,
             Quantity = quantity,
-            Price = price,
+            Price = Math.Round(price, 2, MidpointRounding.ToPositiveInfinity),
```

Comment:
This changes how the middle value is rounded. We could have sth like `Math.Ceil(price * 100) / 100` to round the last digit up. Same thing with buy actions (but with `Floor` instead of `Ceil`).